### ISSUE TITLE
Suppress hlint hints when there are GHC errors

### ISSUE THREAD

**Issue by alanz** [haskell/haskell-ide-engine#443]

**Comment by nponeccop** [comment]
Note the hints are already of lower priority after #375 / #378

**Comment by Anrock** [comment]
At least in Neovim + LanguageClient-neovim lints and warnings aren't displayed if errors are present.
@alanz i think this can be closed.
